data/mods/Aftershock/player/professions.json

```diff
@@ -525,5 +525,17 @@
       "male": [ "boxer_shorts" ],
       "female": [ "sports_bra", "panties" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "afs_ship_escape",
+    "name": "Escaping Crew",
+    "points": 6,
+    "description": "You awake in your cabin to the sound of emergency sirens and the ship's computer calmly announcing in a posh English voice 'Abandon the vessel immediately.', and then counting down from five minutes.  You've got just enough time to hit a supply locker before you need to activate the escape pod down the hall.",
+    "items": {
+      "both": { "items": [ "spacer_jumpsuit", "spacer_cap", "socks", "boots", "wristwatch" ] },
+      "male": [ "briefs" ],
+      "female": [ "sports_bra", "boy_shorts" ]
+    }
```

Comment:
Why not copy-from the rating in for this one, and only change the name and description, they are more or less the same thing as each other anyways.